File: Baekjoon/11000~11999/[11948]과목선택/solution.js
const fileName = process.platform === "linux" ? "/dev/stdin" : "example.dat";
const inputData = require("fs")
  .readFileSync(fileName)
  .toString()
  .trim()
  .split("\n");

const input = (() => {
  let line = 0;
  return () => inputData[line++];
})();

let low = 100;
let res = 0;

for (let i = 0; i < 4; i++) {
  let point = Number(input());

  low = low > point ? point : low;

  res += point;
}

res -= low;
low = 100;

for (let i = 0; i < 2; i++) {
  let point = Number(input());

  low = low > point ? point : low;

  res += point;
}

res -= low;

console.log(res);

File: Baekjoon/11000~11999/[11948]과목선택/solution.py
import sys

low = 100
res = 0

for i in range(4):
    point = int(sys.stdin.readline())

    low = point if low > point else low

    res += point

res -= low
low = 100

for i in range(2):
    point = int(sys.stdin.readline())

    low = point if low > point else low

    res += point

res -= low

sys.stdout.write(str(res) + "\n")
